### ISSUE TITLE
Create new page to query a live model

### ISSUE THREAD

**Issue by fozziethebeat** [LAION-AI/Open-Assistant#428]
This new page should query the path made in #425 with a prompt written by the user and then display the results.  The user should be able to request a response from one or more models in a drop down.  Those model IDs can be hard coded for now.

It should also make it easy to reset the view and start over.

**Comment by fozziethebeat** [comment]
The inference server now exists.  Once there's a docker setup, we should use the interactive [text-client](https://github.com/LAION-AI/Open-Assistant/blob/main/inference/text-client/__main__.py) example to prototype live model interaction.

This should be blocked by a feature flag since it won't be ready for real usage for a while and we need to frame how and when users should use it.

**Comment by olliestanley** [comment]
This seems to have been resolved by #1915

**Comment by olliestanley** [comment]
Actually - we can leave this open for now, there are elements of this ticket which are not complete e.g. the dropdown to select model

**Comment by AbdBarho** [comment]
I will close this now, even though we don't have the drop down, I don't think we will need it, and we can always create a new issue.
